src/com/content-event/event-graded-game.js

```diff
@@ -0,0 +1,15 @@
+import {h, Component}					from 'preact/preact';
```

Comment:
(not this line but) This file should be named `event-my-grades`

Reply:
Removed file